### PR TITLE
Feature db

### PR DESCRIPTION
**Pull request by JankauskasJ**
Creates a Database code first.
DB created from classes DBItem, DBItemHistory, ItemsContext

Replaced ItemManager functionality with DBManager

Left ItemManager code.

### CODE REVIEW

File: src/Backend/src/Controller/Controller.cs [hunk 1]
```diff
@@ -36,9 +36,11 @@ public static Receipt ProcessImage(string source)
         /// <returns>A <see cref="List{T}"/> of <see cref="Item"/> representing the cheapest items found</returns>
         public static List<Item> ProcessReceipt(Receipt source)
         {
-            ItemManager manager = ItemManager.GetInstance();
+            /*ItemManager manager = ItemManager.GetInstance();
 
-            List<Item> cheapList = source.Items.Select(item => manager.FindCheapest(item)).ToList();
```
Comment (gedgaudasnikita) [inline src/Backend/src/Controller/Controller.cs]: Don't comment the code, delete it if it is not needed

File: src/Backend/src/Controller/Controller.cs [hunk 2]
```diff
@@ -49,10 +51,12 @@ public static List<Item> ProcessReceipt(Receipt source)
         /// <param name="source">A <see cref="Receipt"/> to save</param>
         public static void SaveReceipt(Receipt source)
         {
-            ItemManager manager = ItemManager.GetInstance();
+            /*ItemManager manager = ItemManager.GetInstance();
```
Comment (gedgaudasnikita) [inline src/Backend/src/Controller/Controller.cs]: Delete the commented lines

File: src/Backend/src/BusinessLogic/DBManager.cs [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                    db.SaveChanges();
+                }
+            }
+        }
+
+        /// <summary>
+        /// WARNING!
+        /// Clears the entire Database
+        /// </summary>
+        public static void ClearDB()
```
Comment (gedgaudasnikita) [inline src/Backend/src/BusinessLogic/DBManager.cs]: We don't need this method

**Comment by JankauskasJ** [comment]
Tried adding Unity Tests, on existing Database, didn't work.

**Pull request merged**